src/setup.cpp

```diff
@@ -38,15 +38,23 @@ BHC_API bool setup(const char *FileRoot, void (*outputCallback)(const char *mess
     bhcParams &params, bhcOutputs &outputs)
 {
     api_okay = true;
+    InitLog(outputCallback);
     params.internal = new PrintFileEmu(FileRoot, outputCallback);
     PrintFileEmu &PRTFile = *(PrintFileEmu*)params.internal;
-    
+
     try {
     
     #ifdef BHC_BUILD_CUDA
     setupGPU();
     #endif
 
+    HSInfo RecycledHS; //Values only initialized once--reused from top to ssp, and ssp to bot
```

Comment:
Is there a reason these variables were moved around in the function? (More below too)

Reply:
not anymore, and I moved them back.